src/components/Header/index.tsx

```diff
@@ -53,16 +54,22 @@ const Header = ({
                 {isMobile ? 'Docs' : 'Documentation'}
               </a>
             </li>
-            <li className="nav__tabs">
-              <a
-                href="https://nodejs.org/en/download/"
-                className="activeStyleTab"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Download
-              </a>
-            </li>
+            {window.location.pathname === '/404/' ? (
```

Comment:
Why are you disabling the Download button in header incase of 404?
Plus this check is invalid if you wish it to apply on 404 page as a 404 page will be rendered on an infinite number of routes, and not just `/404`.

Also, The tests are failing because you're accessing the window object directly, which throws the error during the build process.
You can put a `typeof window === "undefined"` check before using the window object anywhere.

Reply:
The issue said that "download section to be removed from this page since it's not relevant".
Oh! Is it the "Download Node (LTS)" button?

And thank you very much for the above clarification :smiley:

Reply:
Yeah, I also think it was the Hero component that needs to be removed from 404